src/test/resources/Bad.scala

```diff
@@ -0,0 +1,5 @@
+import shapeless._
+
+class Bad {
+  implicit val b@bar@ar = cachedImplicit[Int]
```

Comment:
I guess this is technically allowed now, but it's considered bad style since it doesn't count as an explicit type on the value.